src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -19,7 +16,6 @@
 
 /**
  * The Main Window. Provides the basic application layout containing
- * a menu bar and space where other JavaFX elements can be placed.
```

Comment:
Did you accidentally remove this?

Reply:
No I removed it cause I deleted the menu bar